code/datums/components/crafting/recipes.dm

```diff
@@ -976,3 +976,32 @@
 	category = CAT_MISC
 	tools = list(TOOL_WIRECUTTER)
 	
+/datum/crafting_recipe/iron_pickaxe
+	name = "Iron Pickaxe"
+	result = /obj/item/pickaxe
+	time = 40
+	reqs = list(/obj/item/stack/sheet/iron = 3,
+			/obj/item/stack/sheet/mineral/wood = 2
```

Comment:
Can ashwalkers and stuff even get wood on lavaland?

Reply:
From what I recall, one of the shrooms gives wood, which also means it can be grown. Beyond that, there's always the ore boxes.